javaslang/src-gen/test/java/javaslang/APITest.java

```diff
@@ -25,6 +25,101 @@ public void shouldNotBeInstantiable() {
         AssertionsExtensions.assertThat(API.class).isNotInstantiable();
     }
 
+    //
+    // Alias should return not null.
+    // More specific test for each aliased class implemented in separate test class
+    //
+
+    @Test
+    public void shouldFunction0ReturnNotNull() {
+        assertThat(function0(() -> null)).isNotNull();
```

Comment:
If this can add more significance to test - yes I can, but how `function1` can return something other than `Function1`?